config/routes.rb

```diff
@@ -6,6 +6,7 @@
   namespace :api do
     namespace :v1 do
       resources :books, only: [:index, :show]
+      post 'open_library_requests/search'
```

Comment:
👀 why a post? Use a get and query params not body request. 
Also using post here is strange, you are not creating anything.